src/dotnet/commands/dotnet-help/LocalizableStrings.cs

```diff
@@ -57,5 +57,7 @@ internal class LocalizableStrings
         public const string MsBuildDefinition = "msbuilds a project and all of its dependencies";
 
```

Comment:
Here and in the const above and in a few other places, we're not using periods the end of the phrase or sentence. We might want to make this consistent.